### DISCUSSION TITLE
Could not load Llama model

### DISCUSSION THREAD

**Discussion by madeganesh22** [ggml-org/llama.cpp#3505] [Q&A] [answered]
Hi, I've been using the GGML model, specifically the ggml-gpt4all-j-v1.3-groovy version, and it was working perfectly. However, today, when I attempted to use it again, I encountered an issue. It displayed an error message: 'Could not load Llama model from path: ggml-gpt4all-j-v1.3-groovy.bin.' I tried to research this problem and came across a website that mentioned that Llama.cpp no longer supports GGML models. I'm wondering if this information is accurate.
I also attempted an alternative solution using the GGUF model, but the results were unsatisfactory. I'm hoping to find a solution to this issue, and I would greatly appreciate any assistance or guidance. Thank you:)

**Comment by ianscrivener** [comment, marked as answer]
There are two (2) versions of GGUF. The latest llama.cpp code requires GGUFv2

Perhaps grab the latest llama.cpp code... and a recent GGUFv2 model from TheBloke like Mistral-7B-OpenOrca-GGUF, say [mistral-7b-openorca.Q4_K_M.gguf](https://huggingface.co/TheBloke/Mistral-7B-OpenOrca-GGUF/resolve/main/mistral-7b-openorca.Q4_K_M.gguf)... and you should be good.

**Reply by madeganesh22** [reply]
Thank you ! It works
